RELEASE-NOTES.md

```diff
@@ -1,3 +1,59 @@
+# Riak CS 1.5.4 Release Notes
+
+## Bugs Fixed
+
+- Disable previous riak object after backpressure sleep is triggered
+  [riak_cs/#1041](https://github.com/basho/riak_cs/pull/1041). This
+  change prevents unnecessary siblings increase when backpressure is
+  triggered under high upload concurrency, and uploads are interleaved
+  during the backpressure sleep.
+- Fix wrong path rewrite in S3 API by unnecessary unquote
+  [riak_cs/#1040](https://github.com/basho/riak_cs/pull/1040). For
+  example, in case when '+' is included in path, '+' (plus, `%2B` in
+  quoted representation) in object key has been mistakenly unquoted to
+  become ' ' (whitespace, `%20` in quoted representaiton). This fix
+  also addresses
+  [riak_cs/#910](https://github.com/basho/riak_cs/pull/910) and
+  [riak_cs/#977](https://github.com/basho/riak_cs/pull/977).
+
+## Notes on upgrading
+
+Due to the bug #1040, objects including '+' had been mistakenly
+rewritten their path and replaced with ' '. This had led to unexpected
+overwrite of an object including '+' in its key (e.g. `foo+bar`) by a
+different object with a mostly same name but replaced with ' '
+(e.g. `foo bar`), and vice verca.
```

Comment:
Another symptom (than overwriting) is variance between names of
uploaded objects and result of list objects.

This can be an issue if client application lists object first and
download them in the results (e.g. restore backuped files).